03.asynchronous/callback/callback.js

```diff
@@ -0,0 +1,18 @@
+import sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(":memory:");
+
+const CREATE_TABLE =
+  "CREATE TABLE books (id integer primary key autoincrement, title text not null unique)";
```

Comment:
SQL のキーワードは大文字で書いてください。キーワード部分とそうでない部分の区別をつけやすくするためです。

```suggestion
  "CREATE TABLE books (id INTEGER PRIMARY KEY AUTOINCREMENT, title TEXT NOT NULL UNIQUE)";
```

Reply:
こちら修正しました!!
216fb719a4620883bd9b902685e655bdcba051bb